src/Nox.Refeence.DataLoaders/DataSeedRunner.cs

```diff
@@ -21,7 +25,9 @@ public void Run()
             }
             catch (Exception e)
             {
-                Console.WriteLine($"Exception happened during seeding data {dataSeeder.GetType().Name}. Exception: {e.Message}");
+                _logger.LogError("Exception happened during seeding data {dataSeeder}. Exception: {Message}"
```

Comment:
add parameters instead of message hard code?

Reply:
parameters are used here